cms/templates/js/course-highlights-enable.underscore

```diff
@@ -0,0 +1,12 @@
+<div class="course-highlights-setting">
+<h2 id="highlights-enabled-label" class="status-highlights-enabled-label">
+  <%- gettext('Weekly Highlight Emails') %>
```

Comment:
I was a little late with this comment, but I'd like to change this to "Weekly Course Highlight Messages" for consistency.

Reply:
I suppose it's a tradeoff between consistency with documentation and the UX on the page itself.  After much discussion we chose to phrase it this way for the first iteration of this feature in order to (1) reduce the real estate used on the page since it's within an already busy UI (not in a separate side-bar or within a modal) and (2) surface the fact that we're sending emails so we're most precise with what's happening within this text itself.